tests/integration/lxd/test_launcher.py

```diff
@@ -216,6 +216,15 @@ def test_launch_use_base_instance(get_instance_and_base_instance, instance_name)
     assert instance.exists()
     assert instance.is_running()
 
+    # collect the hostname of the instance
```

Comment:
Ideally, I would add another test that would set and verify hostnames before and after copying them, but I'm running out of time for this sprint.